src/queryEngine.ts

```diff
@@ -48,9 +48,51 @@ const DB_NAMES = [
       {
         index: { fields: ['timestamp'] },
         ddoc: 'timestamp-index',
-        name: 'Timestamp',
+        name: 'timestamp-index',
         type: 'json' as 'json',
         partitioned: true
+      },
+      {
```

Comment:
You could use calls like `makeMangoIndex('isodate-index-p', ['isoDate'], { partitioned: true })` from edge-server-tools to create these entries, but I see that you would then need to upgrade to the modern database-management tools we have in that repo. So maybe this is a project for another day.